File: src/main/java/com/github/nsc/de/shake/parser/node/logical/LogicalTrueNode.java
package com.github.nsc.de.shake.parser.node.logical;

import com.github.nsc.de.shake.lexer.characterinput.position.PositionMap;

public class LogicalTrueNode extends LogicalNode {

    public LogicalTrueNode(PositionMap map) {
        super(map);
    }

    @Override
    public String getOperator() {
        return null;
    }

    @Override
    public String toString() {
        return "true";
    }
}

File: parser/src/commonMain/kotlin/com/github/nsc/de/shake/parser/node/logical/LogicalTrueNode.kt
package com.github.nsc.de.shake.parser.node.logical

import com.github.nsc.de.shake.util.characterinput.position.PositionMap

class LogicalTrueNode(map: PositionMap) : LogicalNode(map) {
    override val operator: String?
        get() = null

    override fun toJson(): Map<String, *> = mapOf("name" to "LogicalTrueNode")
}